components/fieldgroups/healthcare-plan/healthcare-plan-details.schema.json

```diff
@@ -0,0 +1,92 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/healthcare-plan-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Healthcare Plan Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["healthcare_services"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/healthcare-plan"],
+  "description": "Details of a Health Insurance plan provided by an organization.",
+  "definitions": {
+    "plan-details": {
+      "properties": {
+        "xdm:payerID": {
+          "Title": "Payer ID",
+          "type": "string",
+          "description": "Unique identifier of the payer."
+        },
+        "xdm:contractID": {
+          "Title": "Contract ID",
+          "type": "string",
+          "description": "Unique identifier of the contract that is signed by the member."
+        },
+        "xdm:ownerID": {
+          "Title": "Owner ID",
+          "type": "string",
+          "description": "Unique identifier of the Owner of the policy which could be owned by an individual , group or an organization."
+        },
+        "xdm:network": {
+          "Title": "Network",
+          "type": "string",
+          "description": "The insurer-specific identifier for the insurer-defined network of providers to which the beneficiary may seek treatment which will be covered at the 'in-network' rate, otherwise 'out of network' term."
+        },
+        "xdm:affiliation": {
+          "Title": "Affiliation",
+          "type": "string",
+          "description": "An affiliation to a government service, such as the army or navy."
+        },
+        "xdm:coverageType": {
+          "Title": "Coverage Type",
+          "type": "string",
+          "enum": ["Medical", "Dental", "Vision", "Accident"],
+          "meta:enum": {
+            "Medical": "Medical",
+            "Dental": "Dental",
+            "Vision": "Vision",
+            "Accident": "Accident"
+          },
+          "description": "Member plan coverage type."
+        },
+        "xdm:coverageStartDate": {
```

Comment:
Coverage Start and End Dates are Member-Plan specific These dates are already in the Member==>plans[] array so, we can remove from here.